### ISSUE TITLE
v0.7/0.7.1 does not longer support long types as start/end position

### ISSUE THREAD

**Issue by rolfschr** [daler/pybedtools#154]
I discovered that the following code

```
#!/usr/bin/python
import pybedtools
regions = [('chr2', long(1), long(2), 'tag')]
b = pybedtools.BedTool(regions)
```
does not work on my pybedtools 0.7.1 installation. It works fine with 0.6.9. The following works fine, too:

```
#!/usr/bin/python
import pybedtools
regions = [('chr2', int(1), int(2), 'tag')]
b = pybedtools.BedTool(regions)
```

Now I am not saying this is a bug, but it definetely crashed stuff that was working previously. In my particuar case, the start/end positions come from a database and were ``long`` by default. Running ``str()`` or ``int()`` on the values fixes _my_ problem.

**Comment by daler** [comment]
Thanks for reporting this. Going from 0.6.9 to 0.7.1 there was a major re-write of the Cython code, and looks like this slipped through. Fixed as of 58f6a76.

**Comment by rolfschr** [comment]
Yeah, I read the change log for 0.7 so I suspected that it's the rewrite under the hood. Thanks for fixing this so fast!
